azure-pipelines-pr-validation.yml

```diff
@@ -204,6 +182,25 @@ extends:
             arguments: "-sourceBranchName $(SourceBranchName) -prNumber ${{ parameters.PRNumber }} -commitSHA ${{ parameters.CommitSHA }} -enforceLatestCommit ${{ iif(parameters.EnforceLatestCommit, '1', '0') }}"
           condition: succeeded()
 
+        - task: Powershell@2
```

Comment:
Is the intent that if this fails, at least we already got the PR out for now?

Reply:
(maybe add a comment to that effect)

Reply:
not sure exactly what you mean - I moved it because in order to get the commit details, we need to have it checked out.  I guess I could continue on failure, but I suspect it would fail in followup steps when we try to reset the build number.

 This shouldn't fail anymore since its just reading the local git repo and not making a github API request.